pkg/helmcertifier/checks/checks_test.go

```diff
@@ -132,3 +132,39 @@ func TestContainsTest(t *testing.T) {
 		})
 	}
 }
+
+func TestHasMinKubeVersion(t *testing.T) {
+	type testCase struct {
+		description string
+		uri         string
+	}
+
+	positiveTestCases := []testCase{
+		{description: "minimum Kubernetes version specified", uri: "chart-0.1.0-v3.minkubeversion.tgz"},
```

Comment:
Please use (and update) the same valid chart we've used in other checks; at the end that chart will be a decent representation of how a valid chart according to the default checks will look like.

Reply:
done